src/LinearSolverTypes.f90

```diff
@@ -89,7 +89,10 @@ MODULE LinearSolverTypes
 PUBLIC :: LinearSolverType_Clear_ValidParams
 
 !> set enumeration scheme for TPLs
-INTEGER(SIK),PARAMETER,PUBLIC :: PETSC=0,TRILINOS=1,PARDISO_MKL=2,MKL=3,NATIVE=4
+INTEGER(SIK),PARAMETER,PUBLIC :: PETSC=0,TRILINOS=1,PARDISO_MKL=2,MKL=3,LS_NATIVE=4
```

Comment:
Same here, use the same prefix for all of these

Reply:
Got it. I'll add this in the next commit. If you have ideas for other naming schemes I'm open to them. I've confirmed that leaving the NATIVE temporary causes no problems, so leaving these names as temporaries shouldn't cause any problems either.

Reply:
My preference would be `LINSYS_*` for each of them.  `LS_*` would be ok too, but I tend to prefer slightly longer, more descriptive names.  Just so long as they're all the same

Reply:
I think this still needs to be addressed

Reply:
Names have been updated